domains/transfer-agreements/TransferAgreements.API/API/ApiModels/Responses/TransferAgreementHistoryEntryDto.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using API.Data;
+
+namespace API.ApiModels.Responses;
+
+public record TransferAgreementHistoryEntryDto(
+    long StartDate,
+    long? EndDate,
+    string? SenderName,
+    string? ActorName,
+    string SenderTin,
+    string ReceiverTin,
+    long AuditDate,
```

Comment:
Consider a different word than "AuditDate" here.

Reply:
Changed the name to CreatedAt. The field will reflect when the HistoryEntry itself was created.